### ISSUE TITLE
Administrative docs have 2 different codes

### ISSUE THREAD

**Issue by andrew2net** [relaton/relaton-calconnect#17]
@ronaldtse Do the `CC/A` and `CC/Adm` mean the same type document type in the [docs](https://standards.calconnect.org/administrative/)?
If they are mean the same type, should a reference `CC/A 1601:2016` matches to the ID `CC/Adm 1601:2016`, and should a reference `CC/Adm 1902:2019` matches to the ID `CC/A 1902:2019`?

**Comment by ronaldtse** [comment]
@andrew2net yes, but we should just use one code, CC/A. We do not use CC/Adm anymore.
